### PR TITLE
Added write-ups

### PR DESCRIPTION
**Pull request by Unactived**
needs images files or existing links for a few WU of @clemg

### CODE REVIEW

**Review by lightdiscord — CHANGES_REQUESTED** [review]
Missing changes in SUMMARY.md

**Review by ShiroUsagi-san — DISMISSED** [review]

File: src/SUMMARY.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 - [Phack2021]()
   - [Amuse_bouche]()
     - [WE-1](./Phack2021/Amuse_bouche/WE-1/README.md)
+    - [WE-3](./Phack2021/Amuse_bouche/WE-1/README.md)
```
Comment (lightdiscord) [inline src/SUMMARY.md]: Wrong path

File: src/SUMMARY.md [hunk 2]
```diff
@@ -21,3 +22,37 @@
...
+  - [Steganography]()
+    - [Aliens](./Phack2021/Steganography/Aliens/README.md)
+    - [Alter_Egg-o](./Phack2021/Steganography/Alter_Egg-o/README.md)
+    - [Caumunikassion](./Phack2021/Steganography/Caumunikassion/README.md)
+    - [Chasse_aux_oeufs](./Phack2021/Steganography/Chasse_aux_oeufs/README.md)
+    - [Strong_Daddy](./Phack2021/Steganography/Strong_Daddy/README.md)
+    - [Une_douce_petite_musique](./Phack2021/Steganography/Une_douce_petite_musique/README.md)
+  - [System]()
+    - [Sudoku](./Phack2021/System/Sudoku/README.md)
+    - [To_B,or_!_to_B](./Phack2021/System/To_B,or_!_to_B/README.md)
```
Comment (lightdiscord) [inline src/SUMMARY.md]: It seems special chars are not the best inside the file path, mdbook can't find the correct file

File: src/SUMMARY.md [hunk 3]
```diff
@@ -21,3 +22,37 @@
...
+    - [Sudoku_v2](./Phack2021/System/Sudoku_v2/README.md)
+    - [Graduated](./Phack2021/System/Graduated/README.md)
+  - [Web]()
+    - [Hello_World](./Phack2021/System/Hello_World/README.md)
+    - [Wall-E](./Phack2021/System/Wall-E/README.md)
+    - [VOD](./Phack2021/System/VOD/README.md)
+    - [Fuzz_Me](./Phack2021/System/Fuzz_Me/README.md)
+    - [Harduino](./Phack2021/System/Harduino/README.md)
```
Comment (lightdiscord) [inline src/SUMMARY.md]: Wrong subfolder, `s/System/Web/`

**Review by lightdiscord — CHANGES_REQUESTED** [review]

**Review by lightdiscord — APPROVED** [review]

**Review by ShiroUsagi-san — APPROVED** [review]

**Pull request merged**